src/applications/mhv-supply-reordering/containers/App.jsx

```diff
@@ -32,13 +44,18 @@ App.propTypes = {
   location: PropTypes.shape({
     basename: PropTypes.string,
   }),
+  user: PropTypes.object,
 };
 
+const mapStateToProps = state => ({
+  user: state.user,
```

Comment:
Do we know why we'd do this over using `selectUser`?

Reply:
Is there a preference? I thought it was the same.